test/libsolidity/semanticTests/arithmetics/check_var_init.sol

```diff
@@ -18,4 +18,4 @@ contract D {
 // ----
 // f() -> FAILURE, hex"4e487b71", 0x11
 // g(), 100 wei -> 1
-// gas legacy: 101718
+// gas legacy: 104108
```

Comment:
Yeah, for the record: the ones that increase here seem to be the ones that call constructors. I guess moving reverts to functions increases code size, but the optimizer will inline stuff back and hence ``legacyOptimized`` is not affected?